client/payment-details/timeline/map-events.js

```diff
@@ -201,11 +215,13 @@ const composeFeeString = ( event ) => {
 	}
 
 	return sprintf(
-		/* translators: %1$s is the total fee amount, %2$f%% is the fee percentage, and %3$s is the fixed fee amount. */
-		__( 'Fee (%2$f%% + %3$s): %1$s', 'woocommerce-payments' ),
+		/* translators: %1$s is the total fee amount, %2$f%% is the fee percentage, %3$s is the fixed fee amount
+		 * and %4%s is the fee label  */
+		__( '%4$s (%2$f%% + %3$s): %1$s', 'woocommerce-payments' ),
 		formatCurrency( -feeAmount, feeCurrency ),
 		formatFee( percentage ),
-		formatCurrency( fixed, fixedCurrency )
+		formatCurrency( fixed, fixedCurrency ),
+		isBaseFeeOnly( event ) ? 'Base fee' : 'Fee'
```

Comment:
> __( '%4$s (%2$f%% + %3$s): %1$s', 'woocommerce-payments' ),

While "Base fee" and "Fee" are the only strings that are translatable, I am not quite sure if we should change it here as we include values in other translated strings in this file. Having more context might be more useful?

>  it might make sense to reorder the `sprintf` parameters so it's a little easier to parse.
I agree, updated in https://github.com/Automattic/woocommerce-payments/pull/2928/commits/c16a1522ea05fb65761226ee7e1584c8ffd30c4a

Reply:
@malithsen The `Base fee` and `Fee` strings should probably be localized as well like this:

```JavaScript
isBaseFeeOnly( event ) ? __( 'Base fee', 'woocommerce-payments' ) : _( 'Fee', 'woocommerce-payments' )
```

And since the previous string (`'Fee (%2$f%% + %3$s): %1$s'`) contained a word, it made sense to localize it, but now it doesn't contain any words, only `%`, `+` and parenthesis. I was wondering if in any language that could be different, but I guess that's not the case.

Reply:
ah I misunderstood your original comment. Translating only the fee label makes more sense now. Will update.

Reply:
Updated in https://github.com/Automattic/woocommerce-payments/pull/2928/commits/21a665bf5e7c5ba1537c2aa86d2a2b1c753a0ca2